### DISCUSSION TITLE
Performance issues

### DISCUSSION THREAD

**Discussion by lppedd** [JaneWardSandy/eclipse-pde-partial-idea#21] [General]
![image](https://user-images.githubusercontent.com/19871649/122765662-07f50d80-d2a1-11eb-97d7-6fd68d248e86.png)

We need to think about a way to prevent using 100% of the CPU and all the RAM available.  
I think in my case it depends on the number of bundles, appprox. 5000.
Every time I open up the project I kill the machine, literally.

**Comment by JaneWardSandy** [comment]
Em...It's hard to know what caused this problem, I haven't had the same problem here yet.
Maybe you can use `VisualVM` to see the memory usage and thread?

**Reply by lppedd** [reply]
@JaneWardSandy yeah, I'll take a memory/CPU snapshot.
